<launch>
  <node pkg="ada_feeding_perception" exec="segment_from_point" name="segment_from_point">
    <param from="$(find-pkg-share ada_feeding_perception)/config/segment_from_point.yaml"/>
    <param name="model_dir" value="$(find-pkg-share ada_feeding_perception)/model/"/>
  </node>

</launch>